.gitignore

```diff
@@ -11,6 +11,7 @@
 
 ## vscode local history
 .history/
+.vscode
```

Comment:
we should probably keep .vscoed in the repo

Reply:
Maybe in a different PR

Reply:
well can you at least remove "adding .vscoed to gitignore" from this pr

Reply:
removed in [2a27c8c](https://github.com/Lodestone-Team/client/pull/62/commits/2a27c8c0f3f852b03b9eaf795f873731834e7657)